hw8_sysos/ls.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python3

```

Comment:
Верно, только слитно пишется
```suggestion
#!/usr/bin/env python3

```